src/lib/utils.ts

```diff
@@ -107,8 +111,12 @@ export function processGloss(gloss: any[], targetId: string): ProcessedGloss {
       processedGloss.targetCollection = item.targetCollection;
     } else if (item.targetChapter && item.targetChapter.value) {
       processedGloss.targetChapter = item.targetChapter.value;
+    } else if (item._section && item._section.value) {
```

Comment:
This format gives me a few hives. Is there something dangerous in just filtering for a model and object assigning? Not urgent.